packages/components/src/theme/types.ts

```diff
@@ -0,0 +1,5 @@
+import { SIZES } from './utils/constants';
```

Comment:
I don't see why we need this file in here

Reply:
I changed to types files following the structure used in other files.